drivers/volume/portworx/portworx.go

```diff
@@ -2621,67 +2621,69 @@ func (p *portworx) UpdateMigratedPersistentVolumeSpec(
 	if len(pv.Spec.StorageClassName) != 0 {
 		sc, err := storage.Instance().GetStorageClass(pv.Spec.StorageClassName)
 		if err != nil {
-			return nil, fmt.Errorf("failed in getting the storage class [%v]: %v", pv.Spec.StorageClassName, err)
+			logrus.Warnf("failed in getting the storage class [%v]: %v", pv.Spec.StorageClassName, err)
```

Comment:
Don't we need to return error from here ?

Reply:
No, if the storageclass is not present, we will return error. In that case, we will not add the CSI section. That is the fix